.eslintrc.json

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "eslint-config-edx"
+  "extends": "eslint-config-edx",
+  "globals": {
```

Comment:
I would prefer to fix most of these with per-file exceptions, as our coding style does not allow the use of globals like these in new code. Example: Backbone should not be a defined global variable, although some legacy code expects it to be, I don't want our style guide to encourage that.

The Jasmine and requireJS things are already added due to the environments we've set in our config (`jasmine` and `amd`). ("RequireJS" might not be, I'd have to check. It's not standard.)

The jquery-jasmine stuff is... I don't understand what this library is and why it's adding new globals, and also I thought we removed it? But I guess we should add all of those too, we already have some of the fixture ones.

Ditto adding the Django stuff. And probably `edx` and `XBlock` too. Although, ugh, I really don't like magic globals.

`JSON` we don't need, that's on in the `browser` env of eslint by default.